scripts/runtime_benchmarks.sh

```diff
@@ -41,7 +41,7 @@ echo "Benchmarking pallets for runtime ${runtime}..."
 if [[ $runtime == "development" ]];
 then
   runtime_path="runtime/development"
-  chain="development-local"
+  chain="development"
```

Comment:
Feature creep: Remainder from removing the redundant `development-local` in #1844.